docs/product-getting-started-guide/src/main/asciidoc/task_new-project.adoc

```diff
@@ -1,32 +1,25 @@
 [id='_importing_a_getting_started_business_project']
-= Importing a Getting Started Business Project
+= Creating the traffic violations project
 
-The following chapter instructs you how to import an incomplete business process application. You will extend the application in later chapters.
-
-[IMPORTANT]
-====
-The business process application example includes features that are Technology Preview only. Technology Preview features are not supported with Red Hat production service level agreements (SLAs), might not be functionally complete, and Red Hat does not recommend to use them for production. These features provide early access to upcoming product features, enabling customers to test functionality and provide feedback during the development process.
-For more information on Red Hat Technology Preview features support scope, see https://access.redhat.com/support/offerings/techpreview/.
-====
-
-[float]
-== Prerequisites
-
-* Installed Red Hat JBoss EAP 7.0. For further information, see https://access.redhat.com/documentation/en-us/red_hat_jboss_enterprise_application_platform/7.0/html/installation_guide/[_Red Hat JBoss EAP 7.0 Installation Guide_].
-* Installed {PRODUCT}. For further information, see the {INSTALLATION_GUIDE}.
-* {PRODUCT} is running and you can log in to Business Central with the `admin` role. For further information, see the {INSTALLATION_GUIDE}.
+. Log in to {CENTRAL}.
+. Go to *Menu* -> *Design* -> *Projects*.
++
 
-== Importing the Starter Project
+*Tip*: Red Hat Decision Manager provides a default Space called *myteam*, as shown in the image below. That way a first time user does not need to create one and can proceed with creating a new project.
++
 
-To import the starter project:
+.Default spaces
+image::spaces-teams.png[]
+. Click *Add Project*, expand *Hide Advanced Options*, then enter:
```

Comment:
IBM would say "... then enter the following information:" They don't like incomplete sentences before a colon, nor ending the sentence with "the following:". Don't shoot the messenger.

Reply:
Done and noted for future ref :)